react/features/base/color-scheme/defaultScheme.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import { ColorPalette, getRGBAFormat } from '../styles';
+
+/**
+ * The default color scheme of the application.
+ */
+export default {
+    'features/filmstrip': {
```

Comment:
We always use it when registering anything, why would we make an exception here?

Reply:
That's internal. This is what SDK users will see. It makes for a leaky abstraction.

Reply:
Then we still register the ColorSchemeRegistry with 'features/xxx', but we append the 'features/' part hardcoded in the code?

Reply:
No, sorry, we remove the 'features/' part from the registry, that's how it will be done then. Is it what you want?

Reply:
And can you please explain how it will be a 'leaky abstraction'?

Reply:
That will work, but there is base/ too, which I don't want to see in the SDK.

It's a leaky abstraction because the fact that we have a features/ subdirectory and / or the fact that we call features using "feature/xxx" or "feature/base/yyy" is something internal to the implementation, from the SDK prespective one would be chaning the color of a dialog, "feaures/base/dialog" is weird and it leaks the implementation details.

We might want to have a mapping between ColorScheme keys and feaures, if it comes to it.